ci-build/Dockerfile

```diff
@@ -31,11 +33,16 @@ RUN chmod a+rwx /bin/pdfsizeopt/pdfsizeopt
 ADD https://github.com/validator/validator/releases/download/jar/vnu.jar /whatwg/
 
 # Trying to copy Prince from its DockerHub container like the others does not work; it has too many
-# shared library dependencies. So we get the .deb file from the source:
-ADD https://www.princexml.com/download/prince_13.5-1_debian10_amd64.deb .
-RUN apt-get install --yes --no-install-recommends ./prince_13.5-1_debian10_amd64.deb && \
-    rm prince_13.5-1_debian10_amd64.deb && \
-    echo '@font-face { font-family: serif; src: local("Symbola") }' >> /usr/lib/prince/style/fonts.css
+# shared library dependencies. Additionally, Prince 12 and 13 have bad interactions with pdfsizeopt
+# (https://github.com/pts/pdfsizeopt/issues/145, https://github.com/whatwg/html-build/issues/255),
+# and Prince 11's .deb file only works with older version of Debian. So, we do it this way (plus the
```

Comment:
If this gets out of hand I guess we could put just Prince in its own container. But good as long as that can be avoided.

Reply:
Yeah, I mean, separately there's a whole thing where probably I should be using multiple containers connected with docker-compose, but things seem to be working OK so far.